### ISSUE TITLE
mature

### ISSUE THREAD

**Issue by ceased-ebc** [manasakashyapharinath/Sarcasm-detection-on-twitter-data#1]
is this only factoring in the word mature to calculate sarcasm score?

**Comment by manasakashyapharinath** [comment]
Yes, 'Mature' is considered as the target word to extract the data and calculate the sarcasm score. Because, we found maximum no of tweets with #sarcasm, #sarcastic etc with 'Mature' keyword in it, compared to any other target word we tried.
